bors.toml

```diff
@@ -0,0 +1 @@
+status = [ "bors" ]
```

Comment:
add `delete_merged_branches = true` such that branches on the main repo get removed once they're merged